tests/exampleCourseQuestions.test.js

```diff
@@ -69,7 +69,7 @@ const qidsExampleCourse = [
 describe('Auto-test questions in exampleCourse', function () {
   this.timeout(60000);
 
-  before('set up testing server', helperServer.before(exampleCourseDir));
+  before('set up testing server', helperServer.before(exampleCourseDir, true));
```

Comment:
Rather than a plain boolean, consider an options object as the second argument instead, e.g. `helperServer.before(exampleCourseDir, { allowSyncFailure: true })`.